.github/workflows/labeler.yml

```diff
@@ -0,0 +1,15 @@
+name: PR Labeler
+
+on:
+  - pull_request_target
+
+jobs:
+  pr-label:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/labeler@v4.0.2
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
generic token!

Reply:
@lilx1ao - I'm not sure what your comment implies. Concern over using `secrets.GITHUB_TOKEN`? I'm following the documentation for using that GitHub supplied secret (https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret). You can even see their example on how to use it is actually showing how to use this labeler action too. 🙂

Reply:
ah ok 👍